screens/rewards/RewardsScreen.js

```diff
@@ -33,12 +33,15 @@ export default class RewardsScreen extends React.Component {
       updates: false,
       index: tab,
       routes,
+      isGuest: false,
     };
   }
 
   // Load user record & transactions
   async componentDidMount() {
     const userId = await AsyncStorage.getItem('userId');
+    const isGuest = userId === 'recxEGfvExP4Dv8nr';
```

Comment:
Will eventually maybe have to change this to depend on dev or prod environment but I guess that's a later problem : ) @anniero98